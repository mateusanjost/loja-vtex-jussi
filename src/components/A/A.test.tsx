import TestRenderer from "react-test-renderer";
import AComponent from './index';


describe("AComponent", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<AComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
