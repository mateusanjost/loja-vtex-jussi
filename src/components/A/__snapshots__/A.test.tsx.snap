// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AComponent should match snapshot 1`] = `
<a
  rel="noopener noreferrer"
  target="_blank"
/>
`;
